examples/two-vpcs/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.0.0"
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = "1.45.0"
```

Comment:
the root module has `>= 1.41.1` so do you want to lock this into `1.41.1` like the other example to ensure it works on lowest version in the range?

Reply:
Fxed. Got me to think that perhaps we also should have a check in pre-commits for this condition